iTeChNoLoGiCi/lab1.asm

```diff
@@ -0,0 +1,24 @@
+model small
+.stack 100h      
+.code  
+    x1     db     1  
+    x2     db     3
+    x3     db     2
+    x4     db     4
+start:
+    mov    al,x1

```

Comment:
Indents between instruction and operands. For what?

Reply:
Just to improve readability.

Reply:
Because ASM codestyle requires indents between instructions and operands.
Proof : http://www.sourceformat.com/coding-standard-asm.htm